checks.d/etcd.py

```diff
@@ -68,6 +69,18 @@ def check(self, instance):
         # Load values from the instance config
         url = instance['url']
         instance_tags = instance.get('tags', [])
+
+        # Load the ssl configuration
+        ssl_params = {
+            'ssl_keyfile': instance.get('ssl_keyfile'),
+            'ssl_certfile': instance.get('ssl_certfile'),
+            'ssl_cert_validation': _is_affirmative(instance.get('ssl_cert_validation', True)),
```

Comment:
Actually the default behaviour of the `requests` library is to validate certs. See its [docs] (http://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification):

> By default, verify is set to True.

So the check already validates certs, BC is preserved here.